New package: XRT v0.1.3

JuliaRegistrator opened the pull request:
- Registering package: XRT
- Repository: https://github.com/pc2/XRT.jl
- Created by: @Mellich
- Version: v0.1.3
- Commit: 35bafe713b2238d97fe806b52917a04d48c577ef
- Git reference: main
- Description: Julia wrapper around native XRT C++ API, a runtime for AI Engines and FPGA platforms

goerz commented:
[noblock] Being a wrapper, the name `XRT` seems okay to me, but it will need a manual merge

JuliaHubRegistrator committed:
New package: XRT v0.1.3

UUID: 4d396880-2e3f-4be1-8c36-e87777010d00
Repo: https://github.com/pc2/XRT.jl.git
Tree: eef03c07b26279bd9cede84a3a4a57b621e8d71f

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
